Orange/widgets/evaluate/owconfusionmatrix.py

```diff
@@ -445,7 +446,8 @@ def _isinvalid(x):
 
         # Update the displayed confusion matrix
         if self.results is not None and self.selected_learner:
-            cmatrix = confusion_matrix(self.results, self.selected_learner[0])
+            learner_index = self.selected_learner[0]
+            cmatrix = confusion_matrix(self.results, learner_index)
```

Comment:
In case `self.selected_quantity == 3`, we do not need to compute `cmatrix` (unused), `colsum` (wrong, recomputed later). `rowsum`, heh, is correctly computed here (the computation below should give the same result), but it's computed from `cmatrix`, thus it needs to be computed below (like it is now).